packages/components/tests/dummy/app/templates/components/form/radio-card.hbs

```diff
@@ -0,0 +1,625 @@
+      Labels or instructions are provided when content requires user input.</li>
+    <li><a href="https://www.w3.org/WAI/WCAG21/Understanding/parsing" rel="noopener noreferrer" target="_blank">4.1.1
+        Parsing (A):</a>
+      In content implemented using markup languages, elements have complete start and end tags, elements are nested
+      according to their specifications, elements do not contain duplicate attributes, and any IDs are unique.</li>
+    <li><a
+        href="https://www.w3.org/WAI/WCAG21/Understanding/name-role-value"
+        rel="noopener noreferrer"
+        target="_blank"
+      >4.1.2 Name, Role, Value (A):</a>
+      For all user interface components, the name and role can be programmatically determined; states, properties, and
+      values that can be set by the user can be programmatically set; and notification of changes to these items is
+      available to user agents, including assistive technologies.</li>
+  </ul>
+</section>
+
+<section data-test-percy>
+  <h3 class="dummy-h3" id="showcase"><a href="#showcase" class="dummy-link-section">§</a> Showcase</h3>
+
+  <h4 class="dummy-h4">States</h4>
+  <div class="dummy-form-radio-card-states-grid">
+    {{#each @model.STATES as |state|}}
+      <div>
+        <span class="dummy-text-small">{{capitalize state}}</span>
+        <br />
+        <div mock-state-value={{state}} mock-state-selector="label">
+          <Hds::Form::RadioCard {{on "change" this.onChange}} @disabled={{(eq state "disabled")}} as |R|>
+            <R.Icon @name="hexagon" />
+            <R.Label>Label</R.Label>
+            <R.Description>Description</R.Description>
+          </Hds::Form::RadioCard>
+        </div>
+      </div>
+    {{/each}}
+  </div>
+  <div class="dummy-form-radio-card-states-grid">
+    {{#each @model.STATES as |state|}}
+      <div>
+        <span class="dummy-text-small">{{capitalize state}} selected</span>
+        <br />
+        <div mock-state-value={{state}} mock-state-selector="label">
+          <Hds::Form::RadioCard
+            {{on "change" this.onChange}}
+            @checked="true"
+            @disabled={{(eq state "disabled")}}
+            as |R|
+          >
+            <R.Icon @name="hexagon" />
+            <R.Label>Label</R.Label>
+            <R.Description>Description</R.Description>
+          </Hds::Form::RadioCard>
+        </div>
+      </div>
+    {{/each}}
+  </div>
+
+  <h4 class="dummy-h4">Card content</h4>
+  <Hds::Form::RadioCard::Group @name="radio-card-default" as |G|>
+    <G.Legend>Group legend</G.Legend>
+    {{#each @model.RADIOCARDS as |item|}}
+      <G.RadioCard @checked={{item.checked}} @value={{item.value}} {{on "change" this.onChange}} as |R|>
+        <R.Icon @name="hexagon" />
+        <R.Label>{{item.label}}</R.Label>
+        <R.Badge @text={{item.badge}} />
+        <R.Description>{{item.description}}</R.Description>
+      </G.RadioCard>
+    {{/each}}
+  </Hds::Form::RadioCard::Group>
```

Comment:
Maybe in this section I would do something that stress-test the component in relation to its content: eg. one card with a longer label that spans multiple lines, one card with a multi-line description and one card without badge or icon. This would also allow a visual-check that all the cards have the same height even when the content is so different.

Reply:
was the thumbs up something todo, or just an ACK? on this one I feel more strongly, because it would allow us to test visually (also via visual regression testing) the working of the component

Reply:
I tried to vary the examples used across sections (including different heights) as follows:
How to use:
 - basic example (using the structured subcomponents Icon, Label, Badge, Description with varied lengths on the Label and Description)
 - custom example (using custom width, generic content, and multiple Badges).

Showcase:
 - contains examples with and without badges; can add one without icon – this may be the only one missing from the suggestions above.

Reply:
The issue I have with the "How to use" is that it's not covered by Percy. I would feel more "covered" and at peace of mind if the "variable height/content" example is in the showcase too (only one of the "rows", if you want).

Probably something like this :) :
```
  <span class="dummy-text-small">With different content height</span>
  <Hds::Form::RadioCard::Group @name="radio-card-group-custom" as |G|>
    <G.Legend>Group legend</G.Legend>
    <G.RadioCard @checked="true" {{on "change" this.onChange}} as |R|>
      <R.Icon @name="hexagon" />
      <R.Label>Radio card with normal label</R.Label>
      <R.Badge @text="Badge" />
      <R.Description>This is the radio card description text.</R.Description>
    </G.RadioCard>
    <G.RadioCard {{on "change" this.onChange}} as |R|>
      <R.Icon @name="hexagon" />
      <R.Label>Radio card with long label that spans multiple lines</R.Label>
      <R.Badge @text="Badge" />
      <R.Description>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque erat elit, lacinia at magna
        eget, porttitor lobortis nulla.</R.Description>
    </G.RadioCard>
    <G.RadioCard {{on "change" this.onChange}} as |R|>
      <R.Label>Radio card without icon</R.Label>
      <R.Description>This is the radio card description text.</R.Description>
    </G.RadioCard>
  </Hds::Form::RadioCard::Group>
  ```

Reply:
Valid point about the 'How to use' section – have added the one above, and yes totally agree this example is a good way to prevent layout regression.